RecoEgamma/EgammaTools/interface/Spot.h

```diff
@@ -0,0 +1,35 @@
+#ifndef SPOT_H
+#define SPOT_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h